src/hoi4_world/countries/hoi4_country_converter.cpp

```diff
@@ -226,6 +225,33 @@ hoi4::NameList ConvertNameList(const std::set<std::string>& primary_cultures,
    }
    return name_list;
 }
+std::pair<std::vector<int>, std::set<int>> PickCharacters(const std::vector<int>& source_character_ids,
```

Comment:
For clarity, try

using SpyIds = std::vector<int>;
using LeadersIds = std::set<int>;
std::pair<SpyIds , LeadersIds>